blueprints/app/files/.eslintrc.js

```diff
@@ -57,5 +57,10 @@ module.exports = {
         'node/no-unpublished-require': 'off',
       },<% } %>
     },
+    {
+      // Test files:
+      files: ['tests/**/*'],
```

Comment:
Should this be `tests/**/*-test.js`?

Reply:
Agreed, yes, updated. I think we want to handle both JS and TS files though, so I have updated to:

```
files: ['tests/**/*-test.{js,ts}'],
```

Reply:
yep!